### ISSUE TITLE
Spot lights not working ?

### ISSUE THREAD

**Issue by msnh2012** [ErfanMo77/Syndra#73]

**Comment by ErfanMo77** [comment]
No, currently spotlights don't work. The lighting part is easy, but implementing shadows is a bit difficult. I plan to fix it soon.

**Comment by msnh2012** [comment]
Got it.

**Comment by ErfanMo77** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [fix spotlights](https://trello.com/c/bUifuoWr/95-fix-spotlights)

**Comment by ErfanMo77** [comment]
Fixed here:
https://github.com/ErfanMo77/Syndra/commit/f84a362c0fc1570909cfb81d8c973034c6028049
